### PR TITLE
[TOOLS-2187] Update base image to deb11.

### PR DESCRIPTION
**Pull request by dwelch-spike**
Also adds the "uda" command to wrapper.sh so that uda is reachable.

Testing: built and launched locally, tested --version on each tool.
```
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools aql --version
Aerospike Query Client
Version 7.0.4
C Client Version 6.0.0
Copyright 2012-2021 Aerospike. All rights reserved.
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools asbackup --version
Aerospike Backup Utility
Version 3.11.4
C Client Version 6.0.0
Copyright 2015-2021 Aerospike. All rights reserved.
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools asrestore --version
Aerospike Restore Utility
Version 3.11.4
C Client Version 6.0.0
Copyright 2015-2021 Aerospike. All rights reserved.
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools asinfo --version
Aerospike Information Tool
Version 2.9.1
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools asadm --version
Aerospike Administration Shell
Version 2.9.1
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools asbench --version
asbench version 1.4.0
MBP-D-Welch:aerospike-tools.docker dylanwelch$ docker run -it docker.io/aerospike/aerospike-tools uda --version
uda version 1.0.2
```

### CODE REVIEW

**Review by arrowplum — COMMENTED** [review]
Looks good to me and tested with a few tools. All fine.

**Comment by arrowplum** [comment]
Small caveat. It does install python. I think it wasn't the plan to remove it but just fyi

**Review by arrowplum — APPROVED** [review]

**Review by mcoberly2 — APPROVED** [review]
LGTM
It was mentioned to clean up the dependencies(e.g. python).
Although that might be work for another Jira?

File: Dockerfile [hunk 1]
```diff
@@ -19,14 +19,15 @@ ENV PATH /aerospike:$PATH
 RUN \
   apt-get update -y \
   && apt-get install -y python3-pip python3 python3-distutils python3-apt python wget logrotate ca-certificates python3-dev python3-setuptools openssl python3-openssl libcurl4-openssl-dev\
-  && wget "https://www.aerospike.com/artifacts/aerospike-tools/${AEROSPIKE_VERSION}/aerospike-tools-${AEROSPIKE_VERSION}-debian10.tgz" -O aerospike-tools.tgz \
+  && wget "https://www.aerospike.com/artifacts/aerospike-tools/${AEROSPIKE_VERSION}/aerospike-tools-${AEROSPIKE_VERSION}-debian11.tgz" -O aerospike-tools.tgz \
   && echo "$AEROSPIKE_SHA256 *aerospike-tools.tgz" | sha256sum -c - \
+  && echo sha256sum aerospike-tools.tgz \
```
Comment (jdogmcsteezy) [inline Dockerfile:24]: What's added with this line?

**Review by jdogmcsteezy — APPROVED** [review]
Looks fine just curious about the sha256 sum change

**Comment by dwelch-spike** [comment]
> LGTM It was mentioned to clean up the dependencies(e.g. python). Although that might be work for another Jira?

I think Python is still a dependency because of asloglatency. @jdogmcsteezy is that correct?

File: Dockerfile (hunk 1 above)
Comment (dwelch-spike) [reply Dockerfile]: My bad, that's some debugging I left in there. Removed it

**Comment by dwelch-spike** [comment]
> > LGTM It was mentioned to clean up the dependencies(e.g. python). Although that might be work for another Jira?
> 
> I think Python is still a dependency because of asloglatency. @jdogmcsteezy is that correct?

I talked to Jesse, it is still needed for asloglatency

**Pull request merged**